app/views/organisations/merge_request.html.erb

```diff
@@ -43,6 +43,7 @@
 
     <%= form_for @merge_request, url: merge_requests_path do |f| %>
       <%= f.hidden_field :requesting_organisation_id, value: @organisation.id %>
+            <%= f.hidden_field :status, value: "unsubmitted" %>
```

Comment:
i don't see why we want to expose this on the frontend. can we not just do this on the backend when they `POST` to the `create`?